package/gluon-respondd/files/lib/gluon/respondd/neighbours.cache

```diff
@@ -0,0 +1 @@
+10000
```

Comment:
That's the plan, but I'd like to get some more real-life testing of the code before lowering the cache times, so this will be a post-2016.1 change.